sdk/synapse/synapse-spark/src/models/mappers.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
 /*
  * Copyright (c) Microsoft Corporation.
```

Comment:
delete old copyright lines.  Interesting that these are all generated. Normally we don't lint generated files.

Reply:
Fixed